controllers/territory_controller/territory_controller.py

```diff
@@ -30,6 +32,14 @@ class Claimant(enum.IntEnum):
     ZONE_0 = 0
     ZONE_1 = 1
 
+    @classmethod
+    def zones(cls) -> Tuple[Claimant, ...]:
```

Comment:
I suspect consumers don't need to know this is a tuple -- I'd be tempted to use `Sequence` or even just `Collection` instead.

Reply:
e0eabbecc0093b423489414f1a4bcce1964c9dcf